Sources/Hammer/EventGenerator/HammerLocatable.swift

```diff
@@ -33,3 +33,57 @@ extension String: HammerLocatable {
         return try eventGenerator.viewWithIdentifier(self).windowHitPoint(for: eventGenerator)
     }
 }
+
+/// Creates an absolute offset for a location in screen points.
+public struct OffsetLocation: HammerLocatable {
+    public let location: HammerLocatable?
+    public let x: CGFloat
+    public let y: CGFloat
+
+    /// Creates an offset for a location.
+    ///
+    /// - parameter location: The location to offset. Passing nil will use the default location.
+    /// - parameter x:        The x offset.
+    /// - parameter y:        The y offset.
```

Comment:
Do you want to say in what direction these x/y offsets are expected and relative to what origin point? Or is that implied with this being an iOS-only library?

Reply:
I think that's probably implied from standard iOS coordinates